python/custreamz/setup.cfg

```diff
@@ -0,0 +1,15 @@
+[isort]
```

Comment:
Please copy the other options from the cudf setup.cfg into this file.

Reply:
If I am correct, the flake8 and black tests pass already. Is there a reason why we should explicitly mention those options?

Reply:
I meant specifically the other `isort` options we set for the other libraries, apologies for being unclear.

Reply:
Got it, done!